client/__tests__/e2e/e2e.test.ts

```diff
@@ -444,6 +447,32 @@ describe("graph overlay", () => {
   });
 });
 
+test("zoom limit is 12x", async () => {
```

Comment:
This test colors by and overlays centroid labels to be able to easily get a coordinate overlaying somewhere in the graph.

It then scrolls over that point and gets the test attribute from the graph wrapper to extract the new distance. The scroll was chosen to max out the zoom level, which should equal `scaleMax`.

Reply:
feels like this comment is super helpful for understanding the test! Can we add it to the code please? Thank you!